### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by qsliu2017**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # define env config
 IMG_REPO=kagaya85
-IMG_TAG=jaeger
\ No newline at end of file
+IMG_TAG=jaeger
```
Comment (kagaya85) [inline .env]: please check such deployment config file

File: deployment/kubernetes-manifests/quickstart-k8s/quickstart-ts-deployment-part2.yml [hunk 2]
```diff
@@ -55,16 +55,27 @@ spec:
     spec:
       containers:
       - name: ts-admin-basic-info-service
-        image: codewisdom/ts-admin-basic-info-service:0.0.4
+        image: registry.cn-hangzhou.aliyuncs.com/lavenpok/ts-admin-basic-info-service:sw1.6
```
Comment (Humbertzhang) [inline deployment/kubernetes-manifests/quickstart-k8s/quickstart-ts-deployment-part2.yml:58]: Should use image in docker hub, and should not use image with SkyWalking Agent

File: deployment/kubernetes-manifests/quickstart-k8s/quickstart-ts-deployment-part2.yml [hunk 3]
```diff
@@ -55,16 +55,27 @@ spec:
...
         imagePullPolicy: Always
+        env:
+          - name: NODE_IP
+            valueFrom:
+              fieldRef:
+                fieldPath: status.hostIP
+          - name: SW_AGENT_COLLECTOR_BACKEND_SERVICES
```
Comment (Humbertzhang) [inline deployment/kubernetes-manifests/quickstart-k8s/quickstart-ts-deployment-part2.yml:65]: Those SkyWalking related env should also delete

File: ts-admin-basic-info-service/Dockerfile [hunk 4]
```diff
@@ -1,8 +1,9 @@
-FROM java:8-jre
+FROM registry.cn-hangzhou.aliyuncs.com/lavenpok/skywalking-agent:1.0
```
Comment (Humbertzhang) [inline ts-admin-basic-info-service/Dockerfile:1]: should use `java:8-jre` base image

File: ts-admin-basic-info-service/Dockerfile [hunk 5]
```diff
@@ -1,8 +1,9 @@
...
 
 RUN /bin/cp /usr/share/zoneinfo/Asia/Shanghai /etc/localtime && echo 'Asia/Shanghai' >/etc/timezone
 
 ADD ./target/ts-admin-basic-info-service-1.0.jar /app/
-CMD ["java", "-Xmx200m", "-jar", "/app/ts-admin-basic-info-service-1.0.jar"]
+ENV SW_AGENT_NAME=ts-admin-basic-info-service
+CMD ["java", "-Xmx200m", "-javaagent:/skywalking/agent/skywalking-agent.jar", "-jar", "/app/ts-admin-basic-info-service-1.0.jar"]
```
Comment (Humbertzhang) [inline ts-admin-basic-info-service/Dockerfile:7]: There should also be updated

File: ts-admin-basic-info-service/src/main/resources/logback.xml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (Humbertzhang) [inline ts-admin-basic-info-service/src/main/resources/logback.xml:1]: `grpc-log` appender should not in logback.xml

**Review by Humbertzhang — APPROVED** [review]
LGTM!

**Pull request merged**